src/Nethermind/Nethermind.JsonRpc/IJsonRpcConfig.cs

```diff
@@ -72,7 +72,7 @@ public interface IJsonRpcConfig : IConfig
         
         [ConfigItem(
             Description = "Gas limit for eth_call and eth_estimateGas",
-            DefaultValue = "100000000")]
+            DefaultValue = "25000000")]
```

Comment:
Could you explain why? Is it from eth_call spec?

Reply:
yes, it was in the specification
4.3.1.1 | eth_call MUST consider gas to equal 25 million if the gas parameter is equal to null[!]

Reply:
Ok, but you need to change it in JsonRpcConfig.cs too